src/commands/runtime/action/create.js

```diff
@@ -234,6 +244,9 @@ ActionCreate.flags = {
   sequence: flags.string({
     description: 'treat ACTION as comma separated sequence of actions to invoke' // help description for flag
   }),
+  docker: flags.string({
+    description: 'use provided Docker image (a path on DockerHub) to run the action' // help description for flag
```

Comment:
@rabbah mind insert the word `Restricted !` before `use provided ...` . B/c this feature is not GA, but available to select namespaces only.